giraffe/src/utils/formatters.ts

```diff
@@ -172,10 +172,15 @@ export const timeFormatter = ({
 
   let formatter
 
+  const getValidDate = (d: number): Date => {
+    const date = new Date(d)
```

Comment:
```suggestion
  const getValidDate = (t: number): Date => {
    const date = new Date(t)
```
`t` represents a timestamp here, right? not a date object?

Reply:
Correct.

And to give some context on why this was changed. When we have an empty table, negative Infinity and positive Infinity are passed in as the lower and upper bounds of the time range. Making a new `Date` with Infinity results in an invalid date object. Which in turn leads to a console error during rendering.

Hence, we use the current time as a placeholder for empty tables.

Reply:
Good call on the variable naming. i updated it to use something more descriptive.